compiler/src/dotty/tools/dotc/typer/Typer.scala

```diff
@@ -3222,21 +3222,23 @@ class Typer extends Namer
         if ((inlined ne tree) && errorCount == ctx.reporter.errorCount) readaptSimplified(inlined)
         else inlined
       }
+      else if tree.symbol.name == nme.f && tree.symbol == defn.StringContext_f then
+        // To avoid forcing StringContext_f when starting compiling StingContex
```

Comment:
typo: "StingContex", also I don't really know what "starting compiling" means.

Reply:
should be just "compiling"